ui/docs/manual/using/integration/frontend.md

```diff
@@ -88,112 +66,108 @@ const status = document.getElementById('status');
 </script>
 ```
 
-Now we are going to create an instance of a `WebAuth` from auth0. This is the
-entity that lets us communicate with the the auth0 service. This `WebAuth`
-instance will take some properties based on how your auth0 client is configured.
+Loading our page up to this point you will see our action link, and clicking
+it will navigate you to the familiar Taskcluster user interface to sign in.
+The URL takes the following query parameters:
 
-```js
-const link = document.getElementById('action-link');
-const status = document.getElementById('status');
-const auth = new auth0.WebAuth({
-  domain: 'auth.mozilla.auth0.com',
-  responseType: 'token id_token',
-  scope: 'taskcluster-credentials openid profile',
-  clientID: '<YOUR AUTH0 CLIENT ID>',
-  audience: '<YOUR AUTH0 CLIENT AUDIENCE>',
-  // This redirect URI is configured with your auth0 client.
-  // For this example, we will pretend that our current HTML
-  // page is going to be served no matter what route is used
-  redirectUri: '<YOUR AUTH0 CLIENT REDIRECT URL>'
-});
-```
+| Parameter | Required | Description |
+| --- | --- | --- |
+| response_type |✓ |The value MUST be "code" for requesting an authorization code, or "token” for requesting an access token. |
+| client_id | ✓ | The registered OAuth client. |
+| redirect_uri | ✓ | The URL to which the browser should be redirected after authorization. |
+| scope | ✓ | | An array of URIs to which the server is allowed to redirect the user. |
+| state | Optional (strongly recommended) | An opaque value used by the client to maintain state between the request and callback.  The authorization server includes this value when redirecting the user-agent back to the client.  The parameter SHOULD be used for preventing cross-site request forgery. |
```

Comment:
Markdown seems not to wrap this:

![image](https://user-images.githubusercontent.com/28673/64359443-5656f080-cfd6-11e9-96ff-c88d0a82e97a.png)

so maybe it makes sense to move some of the description into prose after the table?